src/pages/pages.js

```diff
@@ -0,0 +1,2 @@
+import { HomePage } from "./HomePage/HomePage";
+export {HomePage}
```

Comment:
you can directly export here without importing

Reply:
Yeah will do it next PR